variables.tf

```diff
@@ -2,14 +2,26 @@
 # Input Variables
 ##############################################################################
 
+variable "ibmcloud_api_key" {
+  description = "Enter the IBM Cloud APIkey that's associated with this IBM Cloud account."
+  type        = string
+  sensitive   = true
+}
+
 variable "cluster_id" {
   type        = string
-  description = "Enter Id of the target IBM Cloud Red Hat OpenShift Cluster"
+  description = "Enter Id of the target IBM Cloud Red Hat OpenShift cluster. This cluster ID can be found under the Red Hat OpenShift clusters section."
+  nullable    = false
+}
+
+variable "region" {
```

Comment:
It needs region value. Otherwise, I think following code block throws error.. Even I hadn't included initially when I started working on this..
data "ibm_container_cluster_config" "cluster_config" {
  cluster_name_id = var.cluster_id
  config_dir      = "${path.module}/kubeconfig"
  endpoint_type   = var.cluster_config_endpoint_type != "default" ? var.cluster_config_endpoint_type : null
}

Reply:
@NatarajBTI if you saw an issue with this data lookup, then it means you need to pass the region in the DA [here](https://github.com/terraform-ibm-modules/terraform-ibm-mas/blob/e210f64ab72c0da15fcff47f9df940263da0c497/solutions/existing-cluster/provider.tf#L5-L7):
```
provider "ibm" {
  ibmcloud_api_key = var.ibmcloud_api_key
  region                     =  var.region
}
```

The module itself does not need a region variable exposed